src/components/Popups/ReplyField/PopupList.scss

```diff
@@ -8,17 +8,13 @@
     }
 
     .ba-Popup-content {
-        padding-top: 10px;
+        padding: 10px 0;
         border: solid 1px $bdl-gray-30;
         border-radius: $bdl-border-radius-size;
         box-shadow: 0 1px 2px 0 rgba(0, 0, 0, .05);
     }
 
-    .ba-PopupList-item {
-        padding: 10px;
-    }
-
     .ba-PopupList-prompt {
-        padding-top: 0;
+        padding: 0 10px;
```

Comment:
We should avoid setting styles unnecessarily, since it makes overrides easier if they're needed. So, in this case you may be able to use:

```
padding-left: 10px;
padding-right: 10px;
```